app/models/group.rb

```diff
@@ -0,0 +1,3 @@
+class Group < ApplicationRecord
+  belongs_to :students_group
```

Comment:
the other way around, the group has_many :students_group

Reply:
the group has_many :students_groups (with plural form in the end, right?)

i've changed that

Reply:
right! plural indeed